minecraft_servers/Endernation/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "server_name": "Endernation",
+    "nice_name": "EnderNation",
+    "direct_ip": "Endernation.eu",
+    "server_wildcards": [
+      "%.endernation.eu"
+    ],
+    "supported_languages": [
+      "de",
+      "en"
+    ],
+    "social": {
+      "web": "https://EnderNation.eu",
+      "web_shop": "https://Endernation.eu/shop",

```

Comment:
We really don't want to redirect to such a page. So please remove the line or update the page content.
![Bildschirmfoto 2021-09-07 um 00 32 19](https://user-images.githubusercontent.com/45363287/132263689-bd420839-3f54-4cee-ac1e-232c707408dc.png)